hack/e2e.sh

```diff
@@ -21,3 +22,17 @@ if [ -z "$SUITE" ] || [ "$SUITE" == "clusterreader" ]; then
   (awk '/^  azureClusterReaderKubeconfig:/ { print $2 }' <_data/containerservice.yaml | base64 -d) > _data/_out/azure-cluster-reader.kubeconfig
   go test ./test/e2e -test.v -ginkgo.v -ginkgo.focus="\[AzureClusterReader\]" -ginkgo.noColor -ginkgo.randomizeAllSpecs -tags e2e -kubeconfig ../../_data/_out/azure-cluster-reader.kubeconfig 2>&1 | tee azure-reader.log
 fi
+
+if [ -z "$SUITE" ] || [ "$SUITE" == "dedicated-cluster-admin" ]; then
+  echo "Running azure dedicated-cluster-admin tests"
+  fqdn=$(awk '/^  fqdn:/ { print $2 }' <_data/containerservice.yaml)
+  export KUBECONFIG_DEDICATED_ADMIN=_data/_out/dedicated-cluster-admin.kubeconfig
+  export KUBECONFIG_DEDICATED_READER=_data/_out/dedicated-cluster-reader.kubeconfig
+  export KUBECONFIG_NORMAL_USER=_data/_out/normal.kubeconfig
+
+  # oc login is going to create the kubeconfig with the proper permissions
+  KUBECONFIG=$KUBECONFIG_DEDICATED_ADMIN oc login $fqdn --username dedicated-cluster-admin --password "$(awk '/^  dedicatedAdminPasswd:/{ print $2 }' <_data/containerservice.yaml)" --insecure-skip-tls-verify=true
+  KUBECONFIG=$KUBECONFIG_DEDICATED_READER oc login $fqdn --username dedicated-cluster-reader --password "$(awk '/^  dedicatedReaderPasswd:/{ print $2 }' <_data/containerservice.yaml)" --insecure-skip-tls-verify=true
+  KUBECONFIG=$KUBECONFIG_NORMAL_USER oc login $fqdn --username normal --password "$(awk '/^  normalUserPasswd:/{ print $2 }' <_data/containerservice.yaml)" --insecure-skip-tls-verify=true
+  go test ./test/e2e -test.v -ginkgo.v -ginkgo.focus="\[DedicatedAdmin\]" -ginkgo.noColor -ginkgo.randomizeAllSpecs -tags e2e -kubeconfig ../../_data/_out/normal.kubeconfig 2>&1 | tee dedicated-cluster-admin.log
```

Comment:
We can print all logs from this test now w/o the need to use `tee`. You can remove the second command in the pipe.